src/cursor/cur_hs.c

```diff
@@ -1181,20 +1181,22 @@ __curhs_update(WT_CURSOR *cursor)
  *     Discard a cursor range from the history store tree.
  */
 static int
-__curhs_range_truncate(WT_CURSOR *start, WT_CURSOR *stop)
+__curhs_range_truncate(WT_TRUNCATE_INFO *trunc_info)
 {
     WT_CURSOR *start_file_cursor, *stop_file_cursor;
     WT_SESSION_IMPL *session;
 
-    session = CUR2S(start);
-    start_file_cursor = ((WT_CURSOR_HS *)start)->file_cursor;
-    stop_file_cursor = stop != NULL ? ((WT_CURSOR_HS *)stop)->file_cursor : NULL;
+    session = trunc_info->session;
+    stop_file_cursor = NULL;
+
```

Comment:
This whitespace seems unnecessary

Reply:
Also alphabetize the lines to set start first.